src/screens/AppTileScreen.test.tsx

```diff
@@ -36,7 +37,11 @@ beforeEach(() => {
 });
 
 test('renders webview with source prop', () => {
-  render(<AppTileScreen navigation={navigation} route={route} />);
```

Comment:
This screen ends up calling `useAppConfig`. Now, it needs a `QueryClientProvider`, otherwise it throws.